google_forms_to_jira/program.py

```diff
@@ -1,59 +1,69 @@
-"""This program polls a Google Form for new responses and creates a Jira issue for each new response.
+"""Create Jira issues based on Google Forms responses.
 
-Workflow:
-1. Trigger: HTTP GET request.
-2. Poll Forms: Poll the Google Form for new responses.
-3. Create Jira Issue: For each new response, create a Jira issue with the response data.
+Atlassian Jira API documentation:
+- https://docs.autokitteh.com/integrations/atlassian/jira/python
+
+Google Forms API documentation:
+- https://docs.autokitteh.com/integrations/google/forms/events
 """
 
-import json
 import os
-import time
 
-import autokitteh
 from autokitteh import google
 from autokitteh.atlassian import jira_client
 
 
-POLL_INTERVAL = os.getenv("POLL_INTERVAL")
+JIRA_PROJECT_KEY = os.getenv("JIRA_PROJECT_KEY")
+
+jira = jira_client("jira_conn")
+
+
+def on_form_response(event):
+    print("Form response submitted:", event.data)
+    response_id = event.data.response.response_id
+
+    forms = google.google_forms_client("forms_conn").forms()
+    questions = forms.get(formId=event.data.form_id).execute().get("items", [])
```

Comment:
this is only loosely related to this PR, but what if we abstracted away these Google functions? They look unnecessarily complicated. so all the user has to call is `forms.get(formId)`

Reply:
I get your point, but Google's Python libraries use what's known as a "builder pattern" for their APIs. In our specific samples you don't see the benefit in that design choice, so it looks like unnecessary complexity, but if we were to wrap it as you suggest we would limit the functionality of the library, specifically we'd hide the "customizability" of Google's functions.

Also, writing these wrappers would be like our Go wrappers for Starlark: it would be custom code that people are not familiar with, but we'd have to invest a lot of work into creating and maintaining it.